git/essentials/branching/fetching-a-remote-branch.md

```diff
@@ -7,6 +7,11 @@ levels:
 
 type: normal
 
+aspects:
+  
+  - introduction
+  - deep
```

Comment:
I wouldn't qualify this as a deep aspect, you'll need to use this with the basic interaction of git.

Reply:
Definitely workout worthy though?